docker/docker-compose.pr-tests.yml

```diff
@@ -94,10 +119,9 @@ services:
       valkey: { condition: service_healthy }
       rabbitmq: { condition: service_healthy }
     restart: unless-stopped
-    ports: [ "8091:8091" ]
     environment:
-      EDITOAST_PORT: 8091
-      ROOT_URL: "http://localhost:8091"
+      EDITOAST_PORT: 8090
+      ROOT_URL: "http://osrd-editoast-pr-tests:8090"
```

Comment:
pr-tests ports cannot overlap the main stack ports

Reply:
For consistency, could we expose nothing but the gateway then?

(*Ideally*, the port shuffling and exposition change would be done in a separate commit, but it's not that important)

Reply:
I also moved the diff in another commit.